### DISCUSSION TITLE
Chose what display to use

### DISCUSSION THREAD

**Discussion by manatactical** [apohl79/audiogridder#1178] [Ideas]
Hi,

That would be awesome to have the possibility to set up what display to use to show the plugin frame. AudioGridder is on a one-display server but I'm using the screen for something else; so I'm using a display dummy software to avoid having twice the plugin on my monitors.

AudioGridder is using my real display, so I made the dummy my main one in macOS to have other software there.

Is that doable? Does it make sense to you?

Cheers

**Comment by apohl79** [comment]
I will add it to the list of feature requests. Cheers!

**Reply by manatactical** [reply]
Cool! You've been very open to my suggestions so far and I must admit it's a blast using your software for months now. Thank you for that! 

Now that I think about it, on the same kind of feature: 
Do you think it is possible also to have a setting to choose the preferred network for the server side? 

I have 3 networks _(1G ethernet for Internet and Aux MIDI, 2.5G for main MIDI + Audio stream, and Wifi for Universal Control+DragDrop Feature)_. Sadly, the first one is used to advertise the server on the network by default.
I have to manually add the IP of my 2.5G and reconnect everything from time to time to use my 2.5G. That's no big deal, but I have not found anything to do that yet.

Keep up the great work. Loving it!
Cheers

**Reply by apohl79** [reply]
regrading the nic, that is on the list already. cheers!

**Comment by manatactical** [comment]
Hi,

I'm making another suggestion related to the display. It would be nice to choose also the frame anchor to have popup windows also visible. With Kontakt, for instance, I have to go fullscreen in order to see the popup, which is time-consuming and can sometimes be confusing as it makes Kontakt look frozen.

![Screen Shot 2023-03-21 at 11 38 44 AM](https://user-images.githubusercontent.com/9537424/226583947-bde6d4f2-409f-4116-adbc-589c4b6391fd.png)

**Reply by apohl79** [reply]
Will add it to the list. Will require some research.
